Makefile

```diff
@@ -29,11 +30,21 @@ LOADER_OBJS = $(OUT)/loader.o $(OUT)/test_loader.o
 $(OUT)/loader: $(LOADER_OBJS)
 	$(CC) -o $@ $(LOADER_OBJS)
 
-$(CHECK_CC_ARCH):
-	@echo "Check cross compiler exist or not"
-	@echo "CROSS_COMPILE_SUFFIX=$(CROSS_COMPILE_SUFFIX)"
+$(CHECK_CC_ARCH)::
+	@echo "Check cross compiler CROSS_COMPILE_SUFFIX=$(CROSS_COMPILE_SUFFIX) exist or not"
+	@echo "If failed, please specify CROSS_COMPILE_SUFFIX"
 	@which $(patsubst check_cc_%,%,$@)$(CROSS_COMPILE_SUFFIX)-gcc
-	@echo "Pass"
+
+# The old version ld will corrupt the global variable array which contain another 
```

Comment:
Rather than say "old version", you should specify known versions (at least ranging; prior to certain version).

Reply:
Got it